zenodo_accessrequests/views/settings.py

```diff
@@ -99,7 +113,7 @@ def index():
 
     # Links
     links = SecretLink.query_by_owner(current_user).filter(
-        SecretLink.revoked_at == None
+        SecretLink.revoked_at is None
```

Comment:
``SecretLink.revoked_at.is_(None)`` otherwise is doesn't give the correct result it seems like